flocx_market/db/sqlalchemy/api.py

```diff
@@ -133,6 +139,11 @@ def bid_get_all_unexpired(context):
         .filter(models.Bid.status != 'expired').all()
 
 
+def bid_get_all_by_status(status):
```

Comment:
I might put in a `context` parameter here (and in `offer_get_all_by_status` for consistency).  This query should eventually be scoped so that it filters by `context.project_id` if the query doesn't come from an admin.